core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -56,6 +56,11 @@ THE SOFTWARE.
         <l:task href="${rootURL}/fingerprintCheck" icon="icon-fingerprint icon-md" title="${%Check File Fingerprint}"/>
       </j:if>
 
+      <!-- These users will access from Manage Jenkins -->
+      <j:if test="${!h.hasAnyPermission(it, app.ADMINISTER_MANAGE_SYSTEM_READ)}">
+        <l:task href="${rootURL}/computer" icon="symbol-computer" title="${%Nodes}" permission="${it.READ}"/>
```

Comment:
the permission check here `it.READ` means that you need either `Item/Read` or `View/Read` permission to be able to see the link. In general not bad though not really protecting the `/computer` url which only requires `Overall/Read`

From an admin point of view it means now 3 clicks to get from the root to the executor status (Manage Jenkins -> Nodes -> Status) , before it was just one click.
And an administrator will much more likely need to go there than any other user (I've been using the direct link from the widget quite often lately). So Maybe have that as a link to everyone and not just non admins.
As a workaround one could use the sidebarlink plugin (or with one more click but then on every page with customizable header plugin)

Another possibility might be to have the widget only for admins (one can also have the factory that adds the widget to views in a plugin and with a config option to limit it to admins)

Reply:
```suggestion
        <l:task href="${rootURL}/computer" icon="symbol-computer" title="${%Nodes}" />
```

Reply:
I also use the direct link but I'm a bit wary of adding it to the View sidepanel as we're trying to remove links from there, and not just clutter it up.

You can get to it from the context menu without much effort:

<img width="663" alt="image" src="https://github.com/jenkinsci/jenkins/assets/21194782/916f27d9-2241-4c91-aef3-18c98ef62e6e">